qiskit/providers/__init__.py

```diff
@@ -392,21 +427,32 @@ def run(self, circuits. **kwargs):
 how many times the circuit is to be executed. The options available for a
 backend are defined using an :class:`~qiskit.providers.Options` object. This
 object is initially created by the
-:obj:`~qiskit.providers.BackendV1._default_options` method of a Backend class.
+:obj:`~qiskit.providers.BackendV2._default_options` method of a Backend class.
 The default options returns an initialized :class:`~qiskit.providers.Options`
 object with all the default values for all the options a backend supports. For
 example, if the backend supports only supports ``shots`` the
-:obj:`~qiskit.providers.BackendV1._default_options` method would look like::
+:obj:`~qiskit.providers.BackendV2._default_options` method would look like::
 
     @classmethod
     def _default_options(cls):
         return Options(shots=1024)
 
+Tou can also set validators on an :class:`~qiskit.providers.Options` object to
```

Comment:
You

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)